molo/forms/forms.py

```diff
@@ -128,29 +128,45 @@ def create_checkboxes_field(self, field, options):
         return CharacterCountMultipleChoiceField(
             widget=forms.CheckboxSelectMultiple, **options)
 
-    # @property
-    # def formfields(self):
-    #     '''
-    #     Override parent function in order to ensure that the
-    #     overridden create_..._field methods are referenced
-    #     instead of the parent class methods.
-    #     '''
-    #     formfields = OrderedDict()
-    #     self.formfields.update({
-    #         'positive_number': self.create_positive_number_field})
-
-    #     for field in self.fields:
-    #         options = self.get_field_options(field)
-
-    #         if field.field_type in self.formfields:
-    #             method = getattr(self,
-    #                              self.formfields[field.field_type].__name__)
-    #             formfields[field.clean_name] = method(field, options)
-    #         else:
-    #             raise Exception(
-    #                       "Unrecognised field type: " + field.field_type)
-
-    #     return formfields
+    def create_email_field(self, field, options):
+        return django.forms.EmailField(**options)
+
+    def create_url_field(self, field, options):
+        return django.forms.URLField(**options)
+
+    def create_number_field(self, field, options):
+        return django.forms.DecimalField(**options)
+
+    FIELD_TYPES = {
+        'singleline': create_singleline_field,
+        'multiline': create_multiline_field,
+        'date': create_date_field,
+        'datetime': create_datetime_field,
+        'email': create_email_field,
+        'url': create_url_field,
+        'number': create_number_field,
+        'positive_number': create_positive_number_field,
+        'dropdown': create_dropdown_field,
+        'radio': create_radio_field,
+        'checkboxes': create_checkboxes_field,
+        'checkbox': create_checkboxes_field,
```

Comment:
This method isn't defined in this file, but it is in molo.surveys. Does it need to be added here like the email and url ones were?